instructors.md

```diff
@@ -6,11 +6,31 @@ subtitle: Instructor's Guide
 
 ## Timing
 
-Leave about 30 minutes at the start of each workshop and another 15 mins 
-at the start of each sesson for technical difficulties like WiFi and 
-installing things (even if you asked students to install in advance, longer if 
+Leave about 30 minutes at the start of each workshop and another 15 mins
+at the start of each sesson for technical difficulties like WiFi and
+installing things (even if you asked students to install in advance, longer if
 not).
 
+
+## Setting up git in RStudio
+
+There can be difficulties linking git to RStudio depending on the operating system and the version of the operating system. To make sure git is properly installed, the learners should go to the Options window.
+
+* **Mac OS X:**
+  * Go RStudio -> Preferences... -> Git/SVN
+  * Check and see whether there is a path to a file in the "Git executable" window. If not, the next challenge is figuring out where Git is located.
+  * In the terminal enter `which git` and you will get a path to the git executable. In the "Git executable" window you may have difficulties finding the directory since OS X hides many of the systems file. If the user presses "Command-Shift-G" that will allow you to enter `/usr/bin/git` or whatever the path is.
```

Comment:
Change from 
"If the user presses "Command-Shift-G" that will allow you to enter `/usr/bin/git` or whatever the path is."
to
"While the file selection window is open, pressing "Command-Shift-G" will pop up a text box where you will be able to enter `/usr/bin/git` or whatever the path to your git executable might be."